lib/scaffold.dart

```diff
@@ -574,6 +585,10 @@ class BackdropScaffoldState extends State<BackdropScaffold>
                 fit: StackFit.expand,
                 children: <Widget>[
                   _buildBackPanel(),
+                  if (isBackLayerConcealed && widget.frontLayerActiveFactor < 1)
+                    Container(
+                        color: widget.drawerScrimColor,
```

Comment:
default `drawerScrimColor` is not set, either add this check in if condition.

- also should we use the same name or should use `backdropScrimColor`?

Reply:
or we already have defined `inactiveOverlayColor`, can that be used?

@WieFel Give your thoughts too.

Reply:
> or we already have defined `inactiveOverlayColor`, can that be used?
> 
> @WieFel Give your thoughts too.

`inactiveOverlayColor` is for the front layer's overlay when inactive. It might differ from the color that one wants to use for the back layer.
Maybe we could use the two names `inactiveFrontLayerOverlayColor` and `inactiveBackLayerOverlayColor`. But those names are quite long :D

Reply:
> default drawerScrimColor is not set, add qa check in if condition.

Providing a default does follow `Drawer`. Could a scrim impinge on other use cases we need to check in L588?  (Note the [`scrim`](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/container.dart#L401) will not draw when `color == null`.  Maybe that's desirable?)  

I'm happy to make the change, if you think it's OK.

- Possible enhancement: [`DrawerController`](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/drawer.dart#L320) actually tweens to this value from `Colors.transparent`.

> also should we use the same name or should use backdropScrimColor?

Totally guessing: it will be unusual to customize both independently.  I suggest waiting for someone to request different values.

Reply:
looks like `color` null check would still be required since the `_buildScrimColor` is initializes with `blackXX` for `null` value.

@mocktutl lets not use `drawerScrimColor`, instead use  `inactiveFrontLayerOverlayColor` (deprecatign `inactiveOverlayColor`, with proper warning message) and `inactiveBackLayerOverlayColor`.

Reply:
@daadu It does feel like separate colors are necessary, since there could be high contrast between the layers.

What do you think about the name "scrim" for consistency with Material (and avoiding confusion with the similar [widget](https://api.flutter.dev/flutter/widgets/Overlay-class.html))?

Reply:
I am ok with "scrim".